src/runtime/TabsController.ts

```diff
@@ -219,10 +280,31 @@ export class TabsController {
             element.matches(Selector.TAB) && element.dataset.diplodocId
                 ? element.closest(Selector.TAB_LIST)
                 : null;
+
         return tabList?.closest(Selector.TABS);
     }
 
+    private areTabsVertical(target: HTMLElement) {
+        const parent = target.parentElement;
+
+        return target.dataset.diplodocVerticalTab || Boolean(parent?.dataset.diplodocVerticalTab);
```

Comment:
You should recheck this behavior very carefully https://github.com/diplodoc-platform/tabs-extension/blob/master/src/runtime/TabsController.ts#L201-L204

Reply:
add this too https://github.com/diplodoc-platform/tabs-extension/blob/master/src/runtime/TabsController.ts#L56